tests/x11/network/yast2_network_use_nm.pm

```diff
@@ -23,7 +24,7 @@ sub run {
     select_console 'x11';
 
     # return if NetworkManager is already configured as system network manager
-    return if ($nm_is_active);
+    return if ($nm_is_active || is_sle);
```

Comment:
I don't understand why you add `is_sle` here, can you please add some description?

Reply:
NetworkManager is already configured on sle, so return directly